src/Cart/Checkout/CardDetails/CardDetails.css

```diff
@@ -0,0 +1,37 @@
+.card-details {
+  padding-top: 5rem;
+  height: 100vh;
+}
+.payment-heading {
+  text-transform: uppercase;
+  font-weight: bold;
+}
+.card-details-form {
+  width: 80%;
+  margin: 2rem auto;
+}
+.card-details-form input {
+  transition: 0.5s ease-in;
+  margin: 0.5rem;
+  font-size: 16px;
```

Comment:
I would suggest to keep the sizing units uniform. You can use rem everywhere as possible. Only for specific cases, you can use `px`-like non-responsive unit.